database/Listing.js

```diff
@@ -4,11 +4,183 @@ mongoose.Promise = global.Promise;
 
 const listingSchema = new mongoose.Schema({
   //TODO: define listing schema
-  // title: String,
-  // author: String,
-  // imageUrl: String,
-  // body: String,
-  // views: {type: Number, default: 0}
+  hostId: Number,
```

Comment:
Wow your module is HUGE) Great job on figuring it all out. A few things to take a look at:

I don't see a unique listing id field (I remember you said you were planning to start with 1001). Not sure if it is missing or if I missed it. Without it we will have to use _id ObjectId auto-generated by Mongodb which is less convenient because of its format.
A small suggestion/ note - I see you have Superhost property in your schema, ideally you wouldn't need to duplicate this data since you already have reference to host table - hostId (so you can get all host related data including superhost property by calling host service API) And it looks like you will need to use host service API anyway since you will also need host name (I see your module shows host name).

Reply:
I'm with you - once I get further down the path of rendering actual data, I'll make sure that I'm getting host info (name, superhost status) from your service instead of duplicating in mine. Thanks for the feedback!